packages/edit-site/src/components/sidebar-navigation-screen-pages/style.scss

```diff
@@ -1,4 +1,9 @@
-.edit-site-sidebar-navigation-screen-pages__see-all {
-	/* Overrides the margin that comes from the Item component */
-	margin-top: $grid-unit-20 !important;
+.edit-site-sidebar-navigation-screen__sticky-section.edit-site-sidebar-navigation-screen__sticky-section {
```

Comment:
Should we move this [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/sidebar-navigation-screen/style.scss)? And probably have a property in `SidebarNavigationScreen` like we have for `content`?

Reply:
For consistency, I've used this sticky CSS over in the individual [page details PR](https://github.com/WordPress/gutenberg/pull/50767) to style the last modified date element — thanks for the pointer @jameskoster — and have moved the style declaration to `packages/edit-site/src/components/sidebar-navigation-screen/style.scss` as per @ntsekouras [comment](https://github.com/WordPress/gutenberg/pull/50630/files#r1203963408)

Reply:
These styles might not be generalisable as-is, especially the `gap`, unless we're sure they're only ever going to be applied to flex containers.

Reply:
I'm going to leave this as is for now, and revisit once #50767 is merged.

Reply:
> I'm going to leave this as is for now, and revisit once https://github.com/WordPress/gutenberg/pull/50767 is merged.

Merged!